packages/test-utils/api-report.md

```diff
@@ -165,6 +166,11 @@ export class MockPermissionApi implements PermissionApi {
   ): Promise<EvaluatePermissionResponse>;
 }
 
+// @alpha
+export const MockPluginProvider: ({
```

Comment:
The `test-utils` package needs to be updated to support the alpha entrypoint. It needs 3 changes in the `package.json`: the added flag to the `build` script, `alpha` in the `files` list, and the entry point added to `publishConfig`.